docs/DEVELOPMENT.md

```diff
@@ -2,11 +2,11 @@
 
 ## Prerequisites
 - Golang - 1.22.x
-- Operator SDK version - 1.36.1
+- Operator SDK version - 1.39.1
 ```
 export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
 export OS=$(uname | awk '{print tolower($0)}')
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.36.1
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.39.1
```

Comment:
We don't need to download the SDK manually since 0a1028a134bdb4458df717a6aaedd7ab49cfbe8a but the documentation wasn't updated accordingly.

Please drop this block code entirely in a preparatory patch.